### PR TITLE
MAT-6961_v2 Display view button for versioned measures

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR

Jira Ticket: [MAT-6961](https://jira.cms.gov/browse/MAT-6961)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/measureLanding/measureList/MeasureList.tsx [hunk 1]
```diff
@@ -247,7 +247,11 @@ export default function MeasureList(props: {
       selected?.measureSet?.acls
     );
     setCanEdit(isSelectedMeasureEditable);
-    setEditViewButtonLabel(isSelectedMeasureEditable ? "Edit" : "View");
+    setEditViewButtonLabel(
+      isSelectedMeasureEditable && selected?.measureMetaData.draft
```
Comment (adongare) [inline src/components/measureLanding/measureList/MeasureList.tsx:251]: can we not use `canEdit` instead?

Comment (RohitKandimalla) [reply src/components/measureLanding/measureList/MeasureList.tsx:251]: canEdit is used in popOver not sure what is happening over there. Will check that

Comment (RohitKandimalla) [reply src/components/measureLanding/measureList/MeasureList.tsx:251]: It looks like canEdit is only used to see if the user can have permission to do few actions, such as versioning or drafting a measure. 
Technically here setCanEdit should be called as hasAccess or something.

Comment (adongare) [reply src/components/measureLanding/measureList/MeasureList.tsx:251]: oh yeah, my bad that is right.

**Review by adongare — APPROVED** [review]

**Pull request merged**